.github/workflows/slack-develop-commit.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  new_push_job:
+    runs-on: ubuntu-latest
+    name: New push to repo
+    steps:
+    - name: Send GitHub trigger payload to Slack Workflow Builder
+      id: slack
+      uses: slackapi/slack-github-action@v1.23.0
+      with:
+        payload: |
+          {
+            "text": "New commit merged to develop. @cc-pwa-kit-team please review and reconcile",
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "<a target="_blank" href="https://github.com/SalesforceCommerceCloud/pwa-kit/commit/${{ github.sha }}">View Commit</a>"
+                }
+              }
+            ]
+          }
+      env:
```

Comment:
That looks like it should work.

The only comment is that we can DRY the code by extracting the Slack step login into its own action and re-use it instead of duplicating code that already exists here:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/8b235fa54f711eb80abd1f3987d6e1f45720fd79/.github/workflows/test.yml#L108-L118

Alternatively, we can use the existing action from a verified creator by filing an SA.
https://github.com/marketplace/actions/slack-send